docs/dev-machine-setup.md

```diff
@@ -138,41 +138,49 @@ set up a Ubuntu-based VM on GCE with nested KVM enablement can be found in GCE
     so you should use a recent Linux distribution image - such as Ubuntu 18
     (used in the commands below), or equivalent.
 
-    ```
+    ```console
     $ FC_VDISK=disk-ub18
     $ FC_IMAGE=ub18-nested-kvm
-    $ gcloud compute disks create ${FC_VDISK}\
+    $ gcloud compute disks create ${FC_VDISK} \
     --image-project ubuntu-os-cloud --image-family ubuntu-1804-lts
-    $ gcloud compute images create ${FC_IMAGE} --source-disk ${FC_VDISK}\
-    --licenses "https://www.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx"\
+    $ gcloud compute images create ${FC_IMAGE} --source-disk ${FC_VDISK} \
+    --licenses "https://www.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx" \
```

Comment:
Adding a space here makes the [markdown style](https://github.com/firecracker-microvm/firecracker/blob/master/tests/integration_tests/style/test_markdown.py) test fail. Seems like a bug or limitation of the [mdl](https://github.com/markdownlint/markdownlint) linter. I suggest to remove the space for the moment. Otherwise, nice detailing of the GCP setup! Will approve once the tests are successful.

Reply:
That's an interesting corner case of mdl as described [here](https://github.com/markdownlint/markdownlint/issues/128#issuecomment-201106113_). The easiest solution will be to just move that particular line to the end of the command so it doesn't end in whitespace.